### DISCUSSION TITLE
Extra and Capital TV Cyprus

### DISCUSSION THREAD

**Discussion by ROBGagn** [iptv-org/community#491] [Q&A]
Capital TV
https://s1.cystream.net/live/capital/playlist.m3u8?wmsAuthSign="token"

Extra TV
https://s1.cystream.net/live/extra/playlist.m3u8?wmsAuthSign="token"

Anybody who can make auto-updated token based links for these 2 channels ?

Thanks.

Rob.

**Comment by ghost** [comment]
I don't know if they work...check it yourself

**Reply by ROBGagn** [reply]
Not working

**Reply by ghost** [reply]
So what happened to them?

**Comment by ROBGagn** [comment]
??? Theses links never worked because they are tokenised. the same way as it is for Alpha, we need to find a way make a repo with this type of token...
